repl.nix

```diff
@@ -7,9 +7,9 @@
 # ../haskell.nix at a99a0942284e63099116a44c4ccdec49be200184
 , haskellNix ? #../haskell.nix
     builtins.fetchTarball {
-        url = "https://github.com/input-output-hk/haskell.nix/archive/74fd7087.tar.gz";
-        # nix-prefetch-url --unpack https://github.com/input-output-hk/haskell.nix/archive/74fd7087.tar.gz
-        sha256 = "1hc5b9jg4nk8iiqnjmk5jacnm2v3alzq218whyfx602srwq5wma1";
+        url = "https://github.com/input-output-hk/haskell.nix/archive/dbc8deae1057776fa6c786c48a35967a2c560358.tar.gz";
+        # nix-prefetch-url --unpack https://github.com/input-output-hk/haskell.nix/archive/dbc8deae1057776fa6c786c48a35967a2c560358.tar.gz
+        sha256 = "1hc5b9jg4nk8iiqnjmk5jacnm2v3alzq218whyfx602srwq5wm00";
```

Comment:
@hamishmack this update seems to be incorrect? The sha256 is `057wjisqvmdfn1wyhdbw0cs5r03vypz2cvh3hdapiz4xi3x2gj67` for `https://github.com/input-output-hk/haskell.nix/archive/dbc8deae1057776fa6c786c48a35967a2c560358.tar.gz` according to nix?